frontend/src/pages/Search.jsx

```diff
@@ -1,7 +1,31 @@
+import { useSearchParams } from "react-router-dom"
+import { useState, useEffect } from "react"
+
+import SaveSearchButton from "../components/SaveSearchButton"
 import OccupationsList from "../components/OccupationsList"
 import spinner from "../images/loadingSpinner.svg"
 
-export default function Search({ searchResults, searchStatus }) {
+export default function Search({
+  searchResults,
+  searchStatus,
+  searchQuery,
+}) {
+  const [searchParams, setSearchParams] = useSearchParams()
+  const [isSaved, setIsSaved] = useState(false)
+
+  useEffect(() => {
+    searchQuery && setSearchParams({ query: searchQuery })
+  }, [])
```

Comment:
My IDE say this dependency array needs searchQuery and setSearchParams

Reply:
done :)